Cargo.toml

```diff
@@ -19,11 +19,11 @@ path = "src/passthrough.rs"
 tokio = { version = "1.17.0", features = [ "full" ] }
 futures-util = { version = "0.3.21" }
 
-zeroconf = { version = "0.10.5", path = "../zeroconf-rs/zeroconf", optional = true }
```

Comment:
This was intentional, you have to use a fork of zeroconf for async support.

https://github.com/zeyugao/zeroconf-rs.git

Reply:
Now it uses that fork. Tested compilation with the zeroconf feature too.